var/spack/repos/builtin/packages/py-mixedhtseq/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import glob
+
+
+class PyMixedhtseq(PythonPackage):
+    """HTSeq for mixed single and paired end reads"""
+
+    homepage = "https://github.com/schae234/MixedHTSeq"
+    url      = "https://github.com/schae234/MixedHTSeq/archive/v0.1.0.tar.gz"
+
+    version('0.1.0', sha256='234689c8743ae2ba7ad13bc1809a5248184a4b8d16112d5413e09164ab67e157')
+
+    depends_on('python@2.5:2.8', type=('build', 'run'))
+    depends_on('py-numpy', type=('build', 'run'))
+    depends_on('py-htseq', type=('build', 'run'))
+    depends_on('py-ipython', type=('build', 'run'))
+    depends_on('py-pandas', type=('build', 'run'))
```

Comment:
I don't see these deps listed in `setup.py`

Reply:
Yeah, these seem to be unlisted but imported in the provided scripts. Here are the imports: [HTSeq](https://github.com/schae234/MixedHTSeq/blob/6282d5da1d8b0f7deeb3761325d9a2759abab101/scripts/count.py#L3), [pandas](https://github.com/schae234/MixedHTSeq/blob/6282d5da1d8b0f7deeb3761325d9a2759abab101/scripts/combine_counts.py#L6), [IPython](https://github.com/schae234/MixedHTSeq/blob/6282d5da1d8b0f7deeb3761325d9a2759abab101/scripts/count.py#L318)